Cargo.toml

```
@@ -66,6 +66,7 @@ sha2 = { version = "0.9", optional = true }
 serde = "1.0"
 serde_derive = "1.0"
 serde_json = "1.0"
+smallvec = ">=0.6.14" 
```

I think that's fine because by default (without using explicit caret or tilde or similar semver requirements or ranges) this falls back to patch-compatible versions and does not pull a separate 1.x version.
```suggestion
smallvec = "0.6.14" 
```

Could you add a comment saying that this dependency here is only to force a version requirement across the entire dependency graph but is not directly used by sccache? In case all other transitive dependencies upgrade to 1.x, so we know that it's safe to remove this. Thanks!